### PR TITLE
Fix webpack link;

### PR DESCRIPTION
**Pull request by bjornbytes**
Fixes a link that was using markdown syntax.

### CODE REVIEW

File: views/index.etlua [hunk 1]
```diff
@@ -206,7 +206,7 @@ print(<span class="hljs-string">"hello world!"</span>)
                 <li><a href="https://github.com/fengari-lua/fengari-web">fengari-web</a>: to use Fengari in the browser as easily as you might use JavaScript</li>
                 <li><a href="https://github.com/fengari-lua/fengari-interop">fengari-interop</a>: a lua library that makes interoperating with JavaScript objects simple, it is already included in fengari-web</li>
                 <li><a href="https://github.com/fengari-lua/fengari-node-cli">fengari-node-cli</a>: Lua CLI but running on top of Node.js</li>
-                <li><a href="https://github.com/fengari-lua/fengari-loader">fengari-loader</a>: to bundle lua code as part of a [webpack](https://webpack.js.org/) build process</li>
+                <li><a href="https://github.com/fengari-lua/fengari-loader">fengari-loader</a>: to bundle lua code as part of a <a href="https://webpack.js.org">webpack</a> build process</li>
```
Comment (daurnimator) [inline views/index.etlua]: Minor nit: dropped the trailing `/` in the link.

**Review by daurnimator — CHANGES_REQUESTED** [review]
Could you prefix your commit message with the file changed?

**Review by daurnimator — APPROVED** [review]

**Comment by daurnimator** [comment]
Thanks! Looks good to merge.

@giann I'm not at home for the next few weeks; could you merge + deploy?

**Comment by daurnimator** [comment]
@bjornbytes why did you close?

**Comment by bjornbytes** [comment]
I was doing some github housekeeping.  I don't mind resubmitting this if you want, given it's a single link change it might be quicker for you to commit directly to master though.

**Comment by daurnimator** [comment]
Rebased+merged onto master now.

**Comment by giann** [comment]
Deployed

**Pull request closed without merging**